docs/source/basics.rst

```diff
@@ -246,14 +213,11 @@ Quantum computing is inherently probabilistic. We often have to repeat the same
 results we need. Sometimes we expect the results to all be the same, such as when we apply no gates, or only an ``X``
 gate. When we prepare a superposition state, we expect probabilistic outcomes, such as a 50% probability measuring 0 or 1.
 
-The number of `shots` (also called `trials`) is the number of times to execute a program at once.
+The number of shots (also called "trials") is the number of times to execute a program at once.
```

Comment:
The "at once" part doesn't sit well with me, because it makes it sound like they run in parallel. Maybe change it to  "... is the number of times a program is executed within a single request."

Reply:
Good idea!